tests/storage/online_resize/conftest.py

```
@@ -31,6 +31,12 @@ def xfail_if_storage_for_online_resize_does_not_support_snapshots(
         pytest.xfail(f"Storage class for online resize '{sc_name}' doesn't support snapshots")
 
 
+@pytest.fixture(scope="module")
+def xfail_if_gcp_storage_class(storage_class_matrix_online_resize_matrix__module__):
+    if [*storage_class_matrix_online_resize_matrix__module__][0] == StorageClassNames.GCP:
```

Ruff (0.14.10)

Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)